linux/build-environment/Dockerfile

```diff
@@ -0,0 +1,68 @@
+FROM ubuntu:focal
+
+LABEL maintainer="d.vedenko@audacityteam.org"
+LABEL description="A build environment to check the builds for the package maintainers"
+LABEL version="3.0"
+
+ENV TZ=Europe/London
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+RUN apt-get update
+
+RUN apt-get install -y \
+    build-essential \
+    python3-minimal \
+    python3-pip \
+    g++-9 \
+    libstdc++-9-dev \
+    cmake \
+    git
```

Comment:
Switching to a Bash script would also allow you to [use arrays](https://github.com/musescore/MuseScore/blob/b30a63ff3822869cf9186bd92e1b9dc6983238a9/build/Linux%2BBSD/portable/Recipe#L48-L68) to avoid endless backslashes. You can also [add comments](https://github.com/musescore/MuseScore/blob/b30a63ff3822869cf9186bd92e1b9dc6983238a9/build/Linux%2BBSD/portable/Recipe#L24) in arrays, unlike with backslashes.